Maquetado/scss/grid.scss

```diff
@@ -0,0 +1,99 @@
+.tech-container-grid {
+  display: grid;
+  grid-template-columns: 1fr 1 fr 1fr;
+  grid-template-rows: auto;
+}
+
+.container-wrapper {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  margin-bottom: 70px;
+  padding: 0 70px;
+}
+
+.tech1 { 
+  border-right: solid 2px $green;
+  display: flex;
+  align-items: center;
+  grid-column: 1/2;
```

Comment:
I'd use grid-area here which is simpler. It would also be better to move techs around

Reply:
same for all of the techs